src/app/controllers/checkout/checkout.controller.ts

```diff
@@ -0,0 +1,210 @@
+module webShoes {
+  'use script';
+  class Product {
+    id         : number;
+    img1       : string;
+    img2       : string;
+    img3       : string;
+    img4       : string;
+    manufactor : string;
```

Comment:
I guess should be `manufacturer` instead of manufactor

What is the correct way @adrianovalente ??